owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/files/CopyRemoteFileOperation.kt

```diff
@@ -68,9 +68,6 @@ class CopyRemoteFileOperation(
             // nothing to do!
             return RemoteOperationResult(ResultCode.OK)
         }
-        if (targetRemotePath.startsWith(sourceRemotePath) && sourceSpaceWebDavUrl == targetSpaceWebDavUrl) {
-            return RemoteOperationResult(ResultCode.INVALID_COPY_INTO_DESCENDANT)
-        }
```

Comment:
Ok, but in the use case we don't have the second part of the condition (`sourceSpaceWebDavUrl == targetSpaceWebDavUrl`), and that's why we cannot copy into descendants in different spaces. The solution would be merging this condition in the use case. But that will be addressed in the new created issue:
- https://github.com/owncloud/android/issues/4293